Android/app/src/main/java/io/github/project_travel_mate/utilities/CurrencyListViewActivity.java

```diff
@@ -40,8 +40,7 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_conversion_listview);
 
         currences_names = new ArrayList<>();
-        mListview = findViewById(R.id.listView);
-        mCurrencySearch = findViewById(R.id.currencySearch);
```

Comment:
you havent initialized this using butterknife?

Reply:
Yes, above onCreate